### PR TITLE
[ISSUE #6970] Add rest api v2 in core module

### LINKED ISSUE #6970: Rest api v2 in core module
Core module for rest api definition. Refer to #6983 

#### NacosClusterController

前缀 /v1/core/cluster 改为 /v2/core/cluster

接口

​　self　　　　　　  get /self 改为 get /nodes/self

　listNodes　　　　入参由 keyword 改为 address，url 不动，为/nodes

​　listSimpleNodes　整合到listNodes接口中，入参为state，该接口特定值为up。整合后返回值为Member集合

​　getHealth　　　　使用 self 接口。使用后返回值为Member

​　report　　　　　 post /report 改为 put /nodes，接口名改为updateNodes，入参改为List

​　switchLookup　　post /switch/lookup 改为 put /lookup，参数类型改为requestBody，接口名改为updateLookup

​　leave　　　　　　post /service/leave 改为 delete /nodes，接口名改为deleteNodes

<br />

#### CoreOpsController

前缀 /v1/core/ops 改为 /v2/core/ops

接口

​　raftOps　　　　　不动

​　idInfo　　　　　　get /idInfo 改为 get /ids，接口名改为listIds

​　setLogLevel　　　put /log 不动，入参类型改为requestBody，接口名改为updateLog

### PR DESCRIPTION
**Pull request by kevinw66**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

fixes #6970

### CODE REVIEW

**Comment by Jennyfriel** [comment]
Good day,

 About this issue 

 Who can know this?? Anyone can do it ! Have any options for every people not fix any problem about access?? #1 find a solution #2 create new account. How?? Let help other much better to back access in using own identity?! Picture, ids, birth certificate, others.

**Comment by KomachiSion** [comment]
It seems that CI can't pass. Can you check and fix it first?

**Comment by kevinw66** [comment]
> It seems that CI can't pass. Can you check and fix it first?

done

File: style/NacosCheckStyle.xml [hunk 1]
```diff
@@ -60,6 +60,7 @@
         <module name="AbbreviationAsWordInName">
             <property name="ignoreFinal" value="false"/>
             <property name="allowedAbbreviationLength" value="1"/>
+            <property name="allowedAbbreviations" value="VO"/>
```
Comment (KomachiSion) [inline style/NacosCheckStyle.xml:63]: Why change check style?

**Comment by Jennyfriel** [comment]
Good Day,

   I'm sorry late reply i have problem in my internet connection and my
phone and also not enough memory. If i fix it or my contribute I can earn.

**Comment by Jennyfriel** [comment]
What name changed?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: style/NacosCheckStyle.xml (hunk 1 above)
Comment (kevinw66) [reply style/NacosCheckStyle.xml:63]: Cause some domain object like DTO、VO cannot pass checkstyle, I add VO here

**Review by KomachiSion — APPROVED** [review]

**Comment by Jennyfriel** [comment]
Okay

**Pull request merged**